filehandling.c

```diff
@@ -0,0 +1,41 @@
+#include<stdio.h>
+
+struct Employee{
+	char employeeName[50];
+	int employeeId;
+};
+
+int main(){
+    FILE *fp = NULL;
+    struct Employee emp[2];
+    int index;
+    for(index = 0; index < 2; index ++){
+	    printf("Enter the employee details\n");
+	    printf("Name:");
+	    scanf("%s", emp[index].employeeName);
+	    printf("Id:");
+	    scanf("%d", &emp[index].employeeId);
+    }
+    
+    fp = fopen("file.bin", "wb");
```

Comment:
Looks good, but would suggest to open binary file in read-write mode. This will reduce opening/closing file multiple times
```suggestion
    fp = fopen("file.bin", "wb+");
```
Also do necessary changes as needed with this .

Reply:
ok ,I will do